### DISCUSSION TITLE
Course play in the mod folder but not appearing in game

### DISCUSSION THREAD

**Discussion by Willbill12** [Courseplay/Courseplay_FS22#3005] [Q&A] [answered]
Hello, I hope someone is able to help me with my issue.

I have farm sim22 and have downloaded courseplay and put it in the mods folder on my computer, like other mods i have installed and work. but when i go into the game courseplay does not appear anywhere in the game, i cant select it when i load a game and it is not in downloaded content like all other mods.

Let me know what I have done wrong,
Thanks
Bill

**Comment by BalticHawk** [comment, marked as answer]
Check if you are using the newest [game version 1.13.1.0](https://www.farming-simulator.com/updates.php?lang=en&country=gb) and newest [CP version 7.4.0.0](https://github.com/Courseplay/Courseplay_FS22/releases/tag/7.4.0.0).
If it still doesn't appear, post your logfile.

**Reply by Willbill12** [reply]
My game seems to be running on gmae version 1.6.00, It wont update though.
I have good internet connection, however it tells me after awhile that all servers are down on the auto update. Any advise?

**Reply by BalticHawk** [reply]
If you are experiencing problems with your automatic update or do not have access to fast internet speeds, please use our eShop to download the latest full version:

https://eshop.giants-software.com/downloads.php

To install the game on Windows 10 / 11, double click the downloaded „img” file and then double click the Setup.exe and follow the installation wizard.

To install the game with the “zip” file, first unpack the downloaded „zip” file into an arbitrary folder and then double click the FarmingSimulator2022.exe.

To install the game on Mac OS X, double click the FarmingSimulator2022ESD_1_13_1_0.dmg and follow the installation wizard.

**Reply by BalticHawk** [reply]
That's for the Giants Version. If you are using Steam, you have to find another way to go.

**Reply by Willbill12** [reply]
yes i have been into that website and have put int the product code i used to download the original game, I am given the option to download the whole game again, is there a way to download just the update?

**Reply by BalticHawk** [reply]
I don't think so. Sry.

**Reply by Tensuko** [reply]
Double click on your game launcher while holding shift, check if you have Version set to default, then verify your gamefiles.
But yes you need to latest Game Version!

**Reply by Willbill12** [reply]
Thanks for the help, With the latest game version it works like normal
